### DISCUSSION TITLE
captions on case

### DISCUSSION THREAD

**Discussion by bestbean** [kachurovskiy/nanoels#63] [General]
nanoELS old case for WM210 has captions, while newer case has no captions. 
Why?

**Comment by kachurovskiy** [comment]
@bestbean due to the fact that we speak different languages or maybe because the author was lazy :) What worked well for me in the past is printing the required words on paper, covering it with packaging tape from the front and double-sided tape from the back. This results in much more readable labels anyway. WDYT?

**Reply by bestbean** [reply]
It's not important, i was just curious.
Captions are nice to have, maybe.
I imagine a grey or white case, so i could fill the captions with black color, than sandimg the case, resulting in nice captions.
But anyway, case is already here, and i know, what the buttons are for.

**Comment by bestbean** [comment]
![nanoELS_BV20](https://user-images.githubusercontent.com/13404673/161392487-70f83e9e-b0d0-4f01-a030-7c0a6a72fac5.JPG)

**Reply by kachurovskiy** [reply]
Nice 👍 Is everything working well?

**Reply by bestbean** [reply]
Oh yes!
Thank you for this fine prepared DIY kit.
